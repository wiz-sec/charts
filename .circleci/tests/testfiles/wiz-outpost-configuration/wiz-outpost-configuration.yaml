agent:
  clientId: client-id
  clientSecret: client-secret
  enabled: true
  annotations:
    key1: value1
    key2: value2
httpProxyConfiguration:
  enabled: false
  name: wiz-http-proxy-configuration
