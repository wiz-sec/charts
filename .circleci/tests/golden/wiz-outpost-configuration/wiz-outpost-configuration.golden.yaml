---
# Source: wiz-outpost-configuration/templates/agent.secret.yaml
apiVersion: v1
kind: Secret
metadata:
  name: wiz-outpost-agent-creds
  labels:
    helm.sh/chart: "GOLDEN_STATIC_VALUE"
    app.kubernetes.io/name: wiz-outpost-configuration
    app.kubernetes.io/instance: release-test
    app.kubernetes.io/version: "0.0.1"
    app.kubernetes.io/managed-by: Helm
  annotations: 
    key1: value1
    key2: value2

stringData:
  clientId: "client-id"
  clientSecret: "client-secret"
